src/elements/FooterCopyrightModal/CopyrightModalWindow.tsx

```diff
@@ -1,36 +1,25 @@
-import React, { ReactNode } from "react";
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+import React from 'react'
 
 type propCopyrightModalWindow = {
```

Comment:
The naming convention for types is `PascalCase` and it better to use postfix to determine usage `CopyrightModalWindowProps`